test/gtest/mhp/slide_view-3.cpp

```diff
@@ -13,6 +13,7 @@ TYPED_TEST(Slide3, suite_works_for_3_processes_only) {
   EXPECT_EQ(dr::mhp::default_comm().size(), 3); // dr-style ignore
 }
 
+// more Slide3 tests are comming, all assume that there are 3 mpi processes
```

Comment:
Thank you for taking care of this comment. Slide view does not support different halo sides now, so no tests can be added. Please remove both comments - old and new one.